packages/types/src/globals.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * This file defines flags and constants that can be modified during compile time in order to facilitate tree shaking
+ * for users.
+ *
+ * "Magic strings" like `__SENTRY_DEBUG__` must be replaced with actual values or safe statements during our build process.
```

Comment:
Should this be `__DEBUG_BUILD__`?

Reply:
Yup thanks! 3d63759a0cfbebaadd98784fb0bca35e8811d60e